polls/settings.py

```diff
@@ -48,45 +33,47 @@ def get_env(key, default=True):
 
 # Application definition
 
-INSTALLED_APPS = (
+INSTALLED_APPS = [
     'corsheaders',
     'polls',
-)
+]
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = [
```

Comment:
MIDDLEWARE_CLASSES was renamed to MIDDLEWARE and the default examples now use a Python list instead of a tuple. Same goes for `INSTALLED_APPS` above.